src/components/TranslationBannerLegal.js

```diff
@@ -0,0 +1,145 @@
+    bottom: 0rem;
+    right: 0rem;
+  }
+`
+
+const StyledBanner = styled.div`
+  padding: 1rem;
+  max-height: 100%;
+  max-width: 600px;
+  background: ${(props) => props.theme.colors.infoBanner};
+  color: ${(props) => props.theme.colors.black300};
+  display: flex;
+  justify-content: space-between;
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 2px 4px 0px;
+  border-radius: 2px;
+  @media (max-width: ${(props) => props.theme.breakpoints.m}) {
+    max-width: 100%;
+    box-shadow: 0px -4px 10px 0px ${(props) => props.theme.colors.text} 10%;
+  }
+`
+
+const BannerContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: ${(props) =>
+    props.isPageRightToLeft ? `flex-end` : `flex-start`};
+  margin: 1rem;
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    margin-top: 2.5rem;
+  }
+`
+
+const BannerClose = styled.div`
+  position: absolute;
+  top: 0;
+  right: ${(props) => (props.isPageRightToLeft ? `auto` : 0)};
+  margin: 1rem;
+`
+
+const BannerCloseIcon = styled(Icon)`
+  cursor: pointer;
+`
+
+const Row = styled.div`
+  display: flex;
+  align-items: center;
+  margin-bottom: 1rem;
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    flex-direction: column-reverse;
+    align-items: flex-start;
+  }
+`
+
+const ButtonRow = styled.div`
+  display: flex;
+  align-items: center;
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+`
+
+const StyledEmoji = styled(Emoji)`
+  padding-top: 0.5rem;
+  @media (max-width: ${(props) => props.theme.breakpoints.s}) {
+    margin-bottom: 1rem;
+  }
+`
+
+const TranslationBannerLegal = ({
+  shouldShow,
+  originalPagePath,
+  isPageRightToLeft,
+}) => {
+  const [isOpen, setIsOpen] = useState(shouldShow)
+
+  useEffect(() => {
+    if (localStorage.getItem("dont-show-translation-legal-banner") === "true") {
+      setIsOpen(false)
+    } else {
+      setIsOpen(shouldShow)
+    }
+  }, [originalPagePath, shouldShow])
+
+  return (
+    <BannerContainer isOpen={isOpen}>
+      <StyledBanner>
+        <BannerContent isPageRightToLeft={isPageRightToLeft}>
+          <Row>
+            <H3>
+              <Translation id="translation-banner-no-bugs-title" />
+              <StyledEmoji ml={"0.5rem"} size={1.5} text=":bug:" />
+            </H3>
+          </Row>
+          <p>
+            <Translation id="translation-banner-no-bugs-content" />
+          </p>
+          <ButtonRow>
+            <ButtonPrimary
+              onClick={() => {
+                localStorage.setItem("dont-show-translation-legal-banner", true)
+                setIsOpen(false)
+              }}
+            >
+              <Translation id="translation-banner-no-bugs-dont-show-again" />
+            </ButtonPrimary>
+          </ButtonRow>
+          <BannerClose
+            onClick={() => setIsOpen(false)}
+            isPageRightToLeft={isPageRightToLeft}
```

Comment:
The cross to close the banner isn't displaying on the correct side on [/ar/ legal pages](https://ethereumorgwebsitedev01-translationbannerlegalpages.gtsb.io/ar/contributing/)

<img width="637" alt="Screenshot 2022-02-28 at 12 46 45" src="https://user-images.githubusercontent.com/62268199/155985985-377dcf25-67bb-4f36-9944-d25be7250438.png">

Strangely, it is working on [non-legal pages.](https://ethereumorgwebsitedev01-translationbannerlegalpages.gtsb.io/ar/developers/tutorials/)

<img width="562" alt="Screenshot 2022-02-28 at 12 49 15" src="https://user-images.githubusercontent.com/62268199/155986189-d923958e-1230-4d51-b4d9-64e97975b09c.png">

Reply:
Woops, small mistake in render function. Fixed.